tests/sentry/relay/config/test_metric_extraction.py

```diff
@@ -902,13 +902,23 @@ def test_get_metric_extraction_config_with_low_cardinality(default_project: Proj
 @django_db_all
 def test_get_metric_extraction_config_with_unicode_character(default_project: Project) -> None:
     with Feature({ON_DEMAND_METRICS_WIDGETS: True}):
-        # This will cause the Unicode bug to be raised
+        # This will cause the Unicode bug to be raised for the current version
         create_widget(["count()"], "user.name:Armén", default_project)
         create_widget(["count()"], "user.name:Kevan", default_project, title="Dashboard Foo")
         config = get_metric_extraction_config(default_project)
         assert config
         assert config == {
             "metrics": [
+                {
+                    "category": "transaction",
+                    "condition": {"name": "event.tags.user.name", "op": "eq", "value": "Armén"},
```

Comment:
The new spec version supports the Unicode case, thus, it now shows up in the config.